src/map/daily_system.cpp

```diff
@@ -11,49 +11,55 @@ namespace daily
     std::vector<uint16> sundries1DialItems;
     std::vector<uint16> sundries2DialItems;
     std::vector<uint16> specialDialItems;
+    std::vector<uint16> gobbieJunk = { 507, 508, 510, 511, 566, 568, 1239, 1868, 2542, 2543, 4539, 4543, 9777, 15203, 18180, 19220 };
```

Comment:
Could you expand this table out and comment what each of these items are?

Reply:
Done, also commented on the Lua side.